AntWars/AI/AILoader.cs

```diff
@@ -38,6 +45,12 @@ public AILoader(String path)
             }
         }
 
+        /// <summary>
+        /// Erstellt eine neue AI instance und setzt die game und player property.
+        /// </summary>
+        /// <param name="game">Das game</param>
+        /// <param name="player">Der player</param>
```

Comment:
Bis auf Fachbegriffe und Klassennamen würde ich alles in Deutsch bevorzugen. Und die Properties sollten großgeschrieben sein.
Ist aber nicht allzu wichtig, finde ich. Muss jetzt nicht unbedingt überall nachgebessert werden.